modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -300,7 +290,7 @@ GrainTracker::trackGrains()
           {
             Moose::out << "Assigning center " << closest_match_idx << " "
                        << center_points[closest_match_idx] << " absolute distance: " << min_centroid_diff << '\n';
-            _unique_grains[closest_match_idx] = _feature_sets[map_num][feature_num];
+            _unique_grains[closest_match_idx] = std::move(_feature_sets[map_num][feature_num]);
```

Comment:
Yes, invalid, it's equivalent to having called release on it, so you can test against it like a Boolean.